week-9/Homework/mandatory/1-practice/2-code-reading.md

```diff
@@ -15,6 +15,10 @@ Take a look at the following code:
 
 Explain why line 4 and line 6 output different numbers.
 
```

Comment:
I changed the PR to be against Manchester3 branch instead of master, ensure to check this as it may cause problems if people started adding comments :)